sdk/keyvault/ci.yml

```diff
@@ -33,6 +33,8 @@ pr:
     exclude:
       - sdk/keyvault/microsoft-azure
 
+
+
```

Comment:
whitespace not needed.

Reply:
Was needed to do some triggering ;) But will remove now.